optuna/_imports.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
+
```

Comment:
This PR is almost LGTM, but could you exclude this change? It doesn't cause any trouble, but it's unrelated to this PR.

Reply:
```suggestion
```